src/analyzer-range-workspace/analyzer-range-checks.service.ts

```diff
@@ -54,14 +54,15 @@ export class AnalyzerRangeChecksService {
     }
 
     // COMPON-54
+    if (!isUpdate) {
     error = await this.duplicateAnalyzerRangeChecks(
       component.componentId,
       analyzerRange,
     );
     if (error) {
       errorList.push(error);
     }
-
+  }
```

Comment:
This close bracket seems to be in the wrong place.  Should come before the if(error) block